packages/telemetry/bin/ingest-slog

```diff
@@ -0,0 +1 @@
+../src/ingest-slog-entrypoint.js
```

Comment:
Is this a symlink? (I think this is how github renders a symlink). Those make me nervous in git repos, mostly for cross-platform compatibility (windows) but also because I think they're just confusing in version control. Is there an easy way to make this not a symlink? If not, use the symlink.

Reply:
Yeah, it's a choice between writing a shell script wrapper or using a symlink.  I used to choose the wrapper, but found later (from @kriskowal's example) that the symlink is more portable to windows.